python/nav/ipdevpoll/dataloader.py

```diff
@@ -40,6 +40,16 @@
 from nav.ipdevpoll.db import django_debug_cleanup, run_in_thread
 import django.db
 
+
+def load_netbox(netbox_id):
```

Comment:
AAh, so that's the way to do it. I had the get first and that didn't work, so I tried this which did work even though it's quite ugly.